core/search/client.py

```diff
@@ -20,7 +20,7 @@ def __init__(
             verify_certs=verify_certs,
             max_retries=3,
             retry_on_timeout=True,
-            timeout=30,
+            timeout=60 * 30,
```

Comment:
Why did you opt for setting the timeout to this value? It seems a bit high

Reply:
Basically because reindexing takes a longgg time on large indices. We reindex because if the user calls `vectorize` after `add_source`, the existing documents/fields won't be vectorized unless you reindex.

Reply:
Ok, makes sense